src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+import { User } from '../../types/User';
+import { TodoUser } from '../../types/TodoUser';
+
+interface Props {
+  users: User[];
+  todos: TodoUser[];
+  onAdd: (newTodo: TodoUser[]) => void;
+}
+
+export const NewTodo: React.FC<Props> = ({ users, todos, onAdd }) => {
+  const [title, setTitle] = useState('');
+  const [select, setSelect] = useState(0);
+
+  const [touched, setTouched] = useState(false);
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [hasSelectError, setHasSelectError] = useState(false);
+
+  const maxId = Math.max(...todos.map(todo => todo.id));
+
+  const reset = () => {
+    setTitle('');
+    setSelect(0);
+    setTouched(false);
+    setHasTitleError(false);
+    setHasSelectError(false);
+  };
+
+  const handleChangeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+
+    if (hasTitleError && event.target.value) {
+      setHasTitleError(false);
+    }
+  };
```

Comment:
[CODE STYLE] - Try to avoid one-letter variable naming. It could be confusing for another developer, who reads your code. Also, it would really hard to find this variable in the code editor using search. So, instead of `(e)` it would be better to use `(event)` or `(changeEvent)`.